### DISCUSSION TITLE
macbook mi-2010. homebrew error to install.

### DISCUSSION THREAD

**Discussion by vic778** [Homebrew/discussions#816] [Everyday usage]
I am installing homebrew but I receive this message
Error : fetching /user/local/homebrew/library/taps/homebrew-core failed!
Error :no available formula or cask with the "node"
curl: (6) coule mot résolve host: rare.githubusercontent. com

**Comment by SMillerDev** [comment]
This is not an advanced issue at all so I changed the category.

---

Can you copy the complete output and the command that you ran?

**Comment by vic778** [comment]
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"

**Reply by SMillerDev** [reply]
And the complete output

**Comment by vic778** [comment]
![Capture d’écran 2021-02-15 à 20 32 06](https://user-images.githubusercontent.com/77842237/107982426-b3791300-6fcc-11eb-8089-f15101d14009.png)
show

**Reply by SMillerDev** [reply]
You don't have an internet connection it seems. Without that the install won't work.
